### PR TITLE
Packetio client

### PR DESCRIPTION
**Pull request by DanG100**

### CODE REVIEW

File: dataplane/standalone/pkthandler/pktiohandler/pktiohandler.go [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+
+	m.queueRead(msg.GetPortId(), doneCh)
+
+	return nil
+}
+
+func (m *PacketIOMgr) queueRead(id uint64, done chan struct{}) {
+	p := m.hostifs[id]
+	go func() {
+		buf := make([]byte, 9100)
```
Comment (guoshiuan) [inline dataplane/standalone/pkthandler/pktiohandler/pktiohandler.go:225]: Is this number based on anything? e.g., jumbo packet?

File: dataplane/standalone/pkthandler/pktiohandler/pktiohandler.go [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+	"time"
+
+	"google.golang.org/genproto/googleapis/rpc/status"
+	"google.golang.org/grpc/codes"
+
+	"github.com/openconfig/lemming/dataplane/forwarding/util/queue"
+	"github.com/openconfig/lemming/dataplane/internal/kernel"
+
+	log "github.com/golang/glog"
+
```
[the remark this answers is not in the document]
Comment (DanG100) [reply dataplane/standalone/pkthandler/pktiohandler/pktiohandler.go:28]: my formatter separates external packages from the internal ones

File: dataplane/standalone/pkthandler/pktiohandler/pktiohandler.go [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+	}
+
+	go func() {
+		for {
+			select {
+			case <-c.Context().Done():
+				return
+			case data := <-m.sendQueue.Receive():
+				if err := c.Send(&fwdpb.PacketIn{Msg: &fwdpb.PacketIn_Packet{Packet: data.(*fwdpb.Packet)}}); err != nil {
+					continue
```
[the remark this answers is not in the document]
Comment (DanG100) [reply dataplane/standalone/pkthandler/pktiohandler/pktiohandler.go:79]: no, generally we don't care if single packets fails to be sent or received.

File: dataplane/standalone/pkthandler/pktiohandler/pktiohandler.go (hunk 1 above)
Comment (DanG100) [reply dataplane/standalone/pkthandler/pktiohandler/pktiohandler.go]: added a todo for this. this is MTU commonly set in the sai calls received

**Review by guoshiuan — APPROVED** [review]

**Pull request merged**